Formula/boost-build.rb

```diff
@@ -1,8 +1,8 @@
 class BoostBuild < Formula
   desc "C++ build system"
   homepage "https://www.boost.org/build/"
-  url "https://github.com/boostorg/build/archive/boost-1.70.0.tar.gz"
-  sha256 "6630adb18e9fdddf354ce16ee7c358fa79aa0ae264da3b5604cbed6769ce84e5"
+  url "https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.bz2"
```

Comment:
I think we need to keep using github's boostorg/build tarballs:

* https://github.com/boostorg/build/archive/boost-1.71.0.tar.gz

Reply:
That's odd, I don't think I changed this.

Reply:
Ah, I copied and pasted between the bump formula and didn't notice that this URL was different.